iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensator.java

```diff
@@ -67,7 +67,10 @@
 
     /**
      * Get the susceptance for the maximum section count.
+     *
+     * @deprecated Use {@link #getbPerSection()} * {@link #getMaximumSectionCount()} instead.
      */
+    @Deprecated
     double getMaximumB();
```

Comment:
To be able to remove this method one day, you have to provide a default implementation.